packages/devtools_app/lib/src/shared/release_notes/release_notes.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter_markdown/flutter_markdown.dart';
 import 'package:http/http.dart' as http;
 
+import '../../../devtools.dart' as devtools;
```

Comment:
import the files you need directly instead of importing devtools.dart, which just exports a bunch of files

Reply:
In this case this file just contains the version of devtools and isn't the devtools package. So I think this one is different from importing the whole package

Reply:
oh whoops you're right. I was thinking it was `devtools_app.dart`.